RUFAS/routines/manure/IO_helpers/manure_manager_config_handler.py

```diff
@@ -110,23 +110,39 @@ def get_manure_handler_config(self, manure_handler_type_name: str) -> ManureHand
             om.add_error(error_title, error_message, info_map)
             raise KeyError(error_title)
 
-    def get_custom_manure_separator_config(self, manure_separator_type_name: str) -> Optional[ManureSeparatorConfig]:
-        """Returns the custom manure separator config for the given manure separator type name, or None if no custom
-        config exists.
+    def get_manure_separator_config(self, manure_separator_name: str) -> Optional[ManureSeparatorConfig]:
+        """
+        Returns the config for the given manure separator name, or None if no separation is to occur.
 
         Parameters
         ----------
-        manure_separator_type_name : str
-            The name of the manure separator type for which to get the custom config.
+        manure_separator_name : str
+            The name of the manure separator type for which to get the config.
 
         Returns
         -------
         Optional[ManureSeparatorConfig]
-            The custom manure separator config for the given manure separator type name, or None if no custom config
-            exists.
+            The manure separator config for the given manure separator type name, or None if the requested config is
+            "none".
+
+        Raises
+        ------
+        KeyError
+            If the specified manure separator config name is not present and is not "none".
 
         """
-        return self.custom_manure_separator_configs.get(ManureSeparatorType.get_type(manure_separator_type_name), None)
+        if manure_separator_name == "none":
+            return None
+        try:
+            return self.manure_separator_configs[manure_separator_name]
+        except KeyError:
+            info_map = {"class": self.__class__.__name__, "function": self.get_manure_separator_config.__name__}
+            error_title = (
+                f"Attempted to use a non-existent manure separator configuration called '{manure_separator_name}'."
+            )
+            error_message = "Raising ValueError."
```

Comment:
I think that the `error_title` should be a brief summary of the error, while the `error_message` should contain the details.
```suggestion
            error_title = "Unknow Manure Separator Name"
            error_message = (
                f"Attempted to use a non-existent manure separator configuration called '{manure_separator_name}'."
            )
```